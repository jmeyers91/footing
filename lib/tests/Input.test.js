"use strict";
var __importDefault = (this && this.__importDefault) || function (mod) {
    return (mod && mod.__esModule) ? mod : { "default": mod };
};
Object.defineProperty(exports, "__esModule", { value: true });
var react_1 = __importDefault(require("react"));
var react_2 = require("@testing-library/react");
var Input_1 = __importDefault(require("../../src/Input"));
describe('Input', function () {
    test('Should match snapshot', function () {
        var container = react_2.render(react_1.default.createElement(Input_1.default, null)).container;
        expect(container.firstChild).toMatchSnapshot();
    });
});
//# sourceMappingURL=Input.test.js.map