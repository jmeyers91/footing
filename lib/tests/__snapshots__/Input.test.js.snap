// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Input Should match snapshot 1`] = `
<div
  class="sc-bwzfXH fDQTld sc-bdVaJa iCRaLp"
>
  <input
    value=""
  />
</div>
`;
